src/main/steps/you-may-not-be-able-to-england-and-wales/content.ts

```diff
@@ -0,0 +1,26 @@
+import { TranslationFn } from '../../app/controller/GetController';
+import { CommonContent } from '../common/common.content';
+
+const en = ({ isDivorce }: CommonContent) => ({
+  title: `You may not be able to ${isDivorce ? 'get a divorce' : 'end your civil partnership'} in England and Wales`,
```

Comment:
these seem to be a copy of another ticket. https://github.com/hmcts/nfdiv-frontend/pull/387